### ISSUE TITLE
UNDE BRY clarification question

### ISSUE THREAD

**Issue by schafnoir** [schafnoir/neonPlantSampling#13]
@kjones13 
I just wanted to clarify that the analyses you did for UNDE, and the output table in the bry-by-site.pdf, include both Distributed and Tower Plots, correct? So if I want to provide D05 with a list of Tower Plots to sample, I'll just need to take the plotIDs you provided, join with plotType from another table, and then filter out the Distributed Plots. Yesno?

**Comment by kjones13** [comment]
Yes, both distributed and tower plots are included for that site. I could re-write it to just print tower plots in the output, but filtering to tower plots only for analysis would have led to n=2.
